frontend/src/api.ts

```diff
@@ -43,6 +43,14 @@ const PROFILE_FROM_PAGE = (
   [0]
 );
 
+// Extract the media item's analytics if it is embedded in the page
+export const ANALYTICS_FROM_PAGE = (
+  Array.from(document.getElementsByTagName('script'))
+  .filter((element: HTMLScriptElement) => element.type === 'application/analytics+json')
+  .map((element: HTMLScriptElement) => JSON.parse(element.text))
+  [0]
+);
+
```

Comment:
I'd still like to have an aspiration of replacing all ``<a>`` tags with ``<Link>`` tags and this moves us in the wrong direction. Could this be re-factored to either a) use the same approach as ChannelView and PlaylistView which automagically fall back to using the API if the resource is not embedded in the page or b) just fetch the analytics via the API? Then we can remove the ``analytics.html`` template and this special case from ``api.ts``.